iceberg/iceberg-handler/src/test/results/positive/truncate_force_iceberg_table.q.out

```diff
@@ -85,7 +85,7 @@ Retention:          	0
 #### A masked pattern was here ####
 Table Type:         	EXTERNAL_TABLE      	 
 Table Parameters:	 	 
-	COLUMN_STATS_ACCURATE	{\"BASIC_STATS\":\"true\"}
+	COLUMN_STATS_ACCURATE	{\"BASIC_STATS\":\"true\",\"COLUMN_STATS\":{\"id\":\"true\",\"value\":\"true\"}}
```

Comment:
could you please clarify how is this change related to the code changed?

Reply:
The iceberg tests failed after I rebased. I don't think this change is related to the code in SemanticAnalyzer.java. HIVE-25276 also changed these test files.